server/controllers/getFoodById/index.js

```diff
@@ -0,0 +1,31 @@
+const Boom = require('@hapi/boom');
+const { getFoodById } = require('../../database/queries');
+
+module.exports = async (req, res, next) => {
+  try {
+    const { user } = req;
+    const { categoryId, foodId } = req.params;
+    if (!user) {
+      throw Boom.unauthorized('you are not logged in');
+    }
+    if (categoryId > 4 || categoryId < 1) {
```

Comment:
Are categories stored in the database? If so, you could query here instead of checking hard-coded ids